### PR TITLE
fix(docs): deleted `placement: 'bottom-start'`

### LINKED ISSUE #693: Docs: Clicking the Framework select causes the page to scroll to the bottom.
As the title says.
Clicking this Select scrolls the current page to the bottom.
Screenshot 2023-03-28 at 20 26 13

### PR DESCRIPTION
**Pull request by 0xATHERIS**
resolves #693
`placement: 'bottom-start'` was causing the page to jump to the bottom. deleting this didn't cause any side effects

### CODE REVIEW

File: website/src/components/navigation/sidebar/FrameworkSelect.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const FrameworkSelect = (props: FrameworkSelect) => {
   return (
     <Select
       defaultValue={defaultValue}
-      positioning={{ gutter: 2, placement: 'bottom-start', sameWidth: true }}
+      positioning={{ gutter: 2, sameWidth: true }}
```
Comment (ChanhyukPark-Tech) [inline website/src/components/navigation/sidebar/FrameworkSelect.tsx:45]: I think it is not related with page's scroll!

this placement means just select options placement 

```tsx
 /**
     * The initial placement of the floating element
     */
    placement?: Placement;
```

**Review by ChanhyukPark-Tech — CHANGES_REQUESTED** [review]

File: website/src/components/navigation/sidebar/FrameworkSelect.tsx (hunk 1 above)
Comment (ChanhyukPark-Tech) [reply website/src/components/navigation/sidebar/FrameworkSelect.tsx:45]: image

when
```tsx
positioning={{ gutter: 2, placement: 'top-start', sameWidth: true }}
```

Comment (0xATHERIS) [reply website/src/components/navigation/sidebar/FrameworkSelect.tsx:45]: Hi just wanted to provide evidance that removing the `placement` stops this from happening.
By default the placement is set to bottom.


https://user-images.githubusercontent.com/112592168/232896094-b7a96185-908d-4f59-b97a-d0603db8c790.mp4

Comment (ChanhyukPark-Tech) [reply website/src/components/navigation/sidebar/FrameworkSelect.tsx:45]: Holy..
Thanks for giving awesome video
Really It solved well..

I don’t know why they are related..(maybe portal?..)

Comment (0xATHERIS) [reply website/src/components/navigation/sidebar/FrameworkSelect.tsx:45]: It does seem related to the Portal
On line 40 of `FrameworkSelect.tsx` there is `const SelectWrapper = props.noPortal ? Fragment : Portal`.
If you replace the SelectWrapper with a Fragment the issue doesn't happen. So it seems that the placement in the Select has a side effect on the Portal.

**Pull request merged**